frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -35,7 +35,7 @@ const login = async (
     }
   } catch (e: unknown) {
     // eslint-disable-next-line no-alert
```

Comment:
One last thing: could you update the eslint comments to reflect the console log instead of the alert?